### PR TITLE
PS-3707 runner types

### PR DESCRIPTION
**Pull request by pivnicek**
https://keboola.atlassian.net/browse/PS-3707

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@ services:
       - STORAGE_API_TOKEN_MASTER
       - RUNNER_COMMAND_TO_GET_HOST_IP
       - STORAGE_API_TOKEN_READ_ONLY
+      - STORAGE_API_TOKEN_FIXED_TYPE
```
Comment (odinuv) [inline docker-compose.yml:32]: I sort of don't understand this name at all :)

File: Tests/Runner/DataLoaderTest.php [hunk 2]
```diff
@@ -481,4 +487,84 @@ public function testBranchMappingEnabled()
         self::assertInstanceOf(Result::class, $storageState->getInputTableResult());
         self::assertInstanceOf(InputFileStateList::class, $storageState->getInputFileStateList());
     }
+
+    public function testFixedTypeTableCreate()
```
Comment (odinuv) [inline Tests/Runner/DataLoaderTest.php:491]: also here, why is it "Fixed" ?

Comment (pivnicek) [reply Tests/Runner/DataLoaderTest.php:491]: I dunno man 😄 .  I'll change it to just TypedTable, so `testTypedTableCreate` ok?

File: docker-compose.yml (hunk 1 above)
Comment (pivnicek) [reply docker-compose.yml:32]: yeah that name is debilny.  how about `STORAGE_API_TOKEN_TYPED_TABLES` ?

File: src/Docker/Runner/DataLoader/DataLoader.php [hunk 3]
```diff
@@ -101,7 +102,7 @@ public function __construct(
             $this->component->getConfigurationFormat()
         );
 
-        $tokenInfo = $this->clientWrapper->getBasicClient()->verifyToken();
+        $this->tokenInfo = $this->clientWrapper->getBasicClient()->verifyToken();
```
[the remark this answers is not in the document]
Comment (pivnicek) [reply src/Docker/Runner/DataLoader/DataLoader.php:105]: ok

File: docker-compose.yml (hunk 1 above)
Comment (odinuv) [reply docker-compose.yml:32]: definitely better, or STORAGE_API_TOKEN_FEATURE_TABLES_DEFINITION ?

Comment (pivnicek) [reply docker-compose.yml:32]: surre, thanks 🍻

**Comment by pivnicek** [comment]
@odinuv I think I got it covered, thanks for check 🍻

**Review by odinuv — APPROVED** [review]
seems legit

**Pull request merged**